config-xml/schema/windup-jboss-ruleset.xsd

```diff
@@ -282,19 +282,15 @@
                 <xs:element ref="quickfix" minOccurs="0" maxOccurs="unbounded"/>
             </xs:sequence>
             <xs:attribute name="title" type="xs:string" use="required"/>
-            <xs:attribute name="severity" type="severity-type" use="optional" />
+            <xs:attribute name="categoryID" type="category-id-type" use="optional" />
```

Comment:
We uses already capital case like afterMethod. So i think going with lowercase or capitalcase is what we have - categoryId or category-id -> choose what you think is better ;)